src/utilities/district.ts

```diff
@@ -34,12 +40,17 @@ export function getQuotientsToVulnerableSeatMap(districtResult: DistrictResult):
  *
  * @returns vulnerable seat by quotient
  */
-export function getMostVulnerableSeatByQuotient(districtResults: DistrictResult[]) {
+export function getMostVulnerableSeatByQuotient(
+    districtResults: DistrictResult[],
+    nationalPartyResults: PartyResult[],
+    districtThreshold: number
+) {
     const vulnerableDistrictSeats: VulnerableDistrictSeat[] = [];
     districtResults.forEach((districtResult) => {
         if (districtResult.districtSeats > 0) {
+            const partyResultMap = createPartyResultMap(districtResult.partyResults);
             vulnerableDistrictSeats.push({
-                ...getVulnerableSeatByQuotient(districtResult),
+                ...getVulnerableSeatByQuotient(districtResult, partyResultMap, districtThreshold),
```

Comment:
Codacy found an issue: ['getVulnerableSeatByQuotient' was used before it was defined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5991602)